### DISCUSSION TITLE
[1896170] Prince of Suburbia - Part 1+2 [本体+DLC]

### DISCUSSION THREAD

**Discussion by huihui03** [pjy612/SteamManifestCache#273] [Share]
本体
`appid: 1896170`
DLC
`appid: 2149280`
keys
```
1896171;b210131310ba6e0161da389a200c7ca8915bdf84aaa88eda3d362f8ed2aa0d82
1896170;d44847814427aa29fd71855ccdc2784f49b9cde31392fd5ef86ca846da8e6efc
2149280;381f8b708871daa38594dffa4e9871100b6c0aa068857d0e7abbc7eadfb6b719
```
[1896170_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13829118/1896170_config.vdf.txt)
[清单.zip](https://github.com/pjy612/SteamManifestCache/files/13829129/default.zip)

**Comment by pjy612** [comment]
虽然本体免费，但DLC收费 所以可以
